backend/smalldata/views.py

```diff
@@ -1,7 +1,19 @@
+import sys
+import time
+
 from rest_framework import viewsets
 from .serializers import UtteranceSerializer, CategorySerializer
 from .models import Utterance, Category
+sys.path.insert(0,'/Users/borisjoens/Dropbox/Kommentare/SmallData/src')
```

Comment:
Mh, this is not optimal. Having an absolute path in the import block will cause the code to run only on your local setup (I don't have the path `/Users/borisjoens/Dropbox/Kommentare/SmallData/src` on my machine). I will think of something here...

Reply:
ja ich weiss nicht, aber ohne das findet er die Module nicht.
Verstehe die Syntax noch nicht, wie man aus anderen Ordnern importiert.

Reply:
Obwohl der Code mit sys.path.insert funktioniert,
sind die Module in PyCharm immer noch als "Unresolved reference" markiert.